app/Http/Controllers/Backend/AuthController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Validator;
+use App\Http\Controllers\Controller;
+use Illuminate\Foundation\Auth\ThrottlesLogins;
+use Illuminate\Foundation\Auth\AuthenticatesAndRegistersUsers;
+use Auth;
+use App\Http\Requests;
+use Illuminate\Http\Request;
+
+class AuthController extends Controller
+{
+    use AuthenticatesAndRegistersUsers, ThrottlesLogins;
+
+    /**
+     * Where to redirect users after login / registration.
+     *
+     * @var string
+     */
+    protected $redirectTo = 'admin/dashboard';
+
+    /**
+     * Create a new authentication controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware($this->guestMiddleware(), ['except' => 'logout']);
+    }
+
+    /**
+     * Function show view login.
+     *
+     * @return view
+     */
+    public function getLogin()
+    {
+        return view('backend.auth.login');
+    }
+
+    /**
+     * Handle a login request to the application.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function postLogin(Request $request)
```

Comment:
This function is already defined in Laravel core, you don't need to define again, just extend from the `Controllers\Auth\AuthController` and modify the properties.